---
title: Zzemp
date: 2025-06-29
author: Your Name
cell_count: 6
score: 5
---

```python
import pyutil as pyu
pyu.get_local_pyinfo()
```




    'conda env: C:\\Users\\Afia Jahan\\anaconda3\\envs\\py312; pyv: 3.12.11 | packaged by Anaconda, Inc. | (main, Jun  5 2025, 12:58:53) [MSC v.1929 64 bit (AMD64)]'




```python
print(pyu.ps2("yfinance pandas matplotlib"))
```

    
    


```python
import yfinance as yf
import pandas as pd
import numpy as np
import matplotlib.pyplot as plt
```


```python
def show_graph(symbol):

pass
```


      Cell In[11], line 3
        pass
        ^
    IndentationError: expected an indented block after function definition on line 1
    



```python
show_graph("AMZN")

```


    ---------------------------------------------------------------------------

    NameError                                 Traceback (most recent call last)

    Cell In[12], line 1
    ----> 1 show_graph("AMZN")
    

    NameError: name 'show_graph' is not defined



```python

```


---
**Score: 5**